packages/ui/build/vite/configs/vite.esm.js

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vite'
+import getViteConfig from '../common-config'
+
+export default ({ mode }) => {
+  const format = 'esm'
```

Comment:
I think here the `format` variable is redundant (in the other configuration files too).